plugins/discovery-gce/build.gradle

```diff
@@ -7,13 +7,14 @@ esplugin {
 }
 
 versions << [
-  'google': '1.23.0'
+  'google': '1.23.0',
```

Comment:
is the client forward compatible with the oauth client? Maybe we should upgrade everything? I haven't looked into detail

Reply:
Good point, I've submitted #85132